xdsl/dialects/arith.py

```diff
@@ -42,6 +42,38 @@
 
 _FloatTypeT = TypeVar("_FloatTypeT", bound=AnyFloat)
 
+_CMPI_COMPARISON_OPERATIONS = [
+    "eq",
+    "ne",
+    "slt",
+    "sle",
+    "sgt",
+    "sge",
+    "ult",
+    "ule",
+    "ugt",
+    "uge",
+]
+
+_CMPF_COMPARISON_OPERATIONS = [
+    "false",
+    "oeq",
+    "ogt",
+    "oge",
+    "olt",
+    "ole",
+    "one",
+    "ord",
+    "ueq",
+    "ugt",
+    "uge",
+    "ult",
+    "ule",
+    "une",
+    "uno",
```

Comment:
Can you make these two constants public (without the `_`)?
In case we ever need to access it from outside.